src/main/java/util/csvparsing/CsvMonumentConverter.java

```diff
@@ -0,0 +1,91 @@
+package util.csvparsing;
+
+import com.monumental.models.Monument;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+/**
+ * Class used to convert a CSV row representing a Monument into a Monument object
+ */
+public class CsvMonumentConverter {
+    /**
+     * Static method for converting a CSV row representing a Monument into a Monument object
+     * @param csvRow - CSV row representing a Monument, as a String
+     * @return Monument - the Monument object represented by csvRow
+     */
+    public static Monument convertCsvRowToMonument(String csvRow) {
+        // Regex to split on commas only if the comma has zero or an even number of quotes ahead of it
+        // See: https://stackoverflow.com/questions/1757065/java-splitting-a-comma-separated-string-but-ignoring-commas-in-quotes
+        String[] csvRowArray = csvRow.split(",(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)", -1);
+        Monument result = new Monument();
+
+        for (int columnIndex = 0; columnIndex < csvRowArray.length; columnIndex++) {
+            String value = csvRowArray[columnIndex];
+
+            // NOTE: The order of the columns is specific to the initial dataset
```

Comment:
Normally with csv you read in the first line as headers

Reply:
Got lazy :)